charts/fluent-operator/templates/fluentbit-clusterinput-systemd.yaml

```diff
@@ -8,11 +9,13 @@ metadata:
     fluentbit.fluent.io/component: logging
 spec:
   systemd:
-    tag: service.*
-    path: /var/log/journal
-    db: /fluent-bit/tail/systemd.db
-    dbSync: Normal
-    systemdFilter:
-      - _SYSTEMD_UNIT={{ .Values.containerRuntime }}.service
-      - _SYSTEMD_UNIT=kubelet.service
-{{- end }}
\ No newline at end of file
+    tag: {{ .Values.fluentbit.input.systemd.tag | default "service.*" }}
+    path: {{ .Values.fluentbit.input.systemd.path | default "/var/log/journal" }}
+    db: {{ .Values.fluentbit.input.systemd.db | default "/fluent-bit/tail/systemd.db" }}
+    dbSync: {{ .Values.fluentbit.input.systemd.dbSync | default "Normal" }}
+    systemdFilter: {{ .Values.fluentbit.input.systemd.systemdFilter | default ["_SYSTEMD_UNIT={{ .Values.containerRuntime }}.service", "_SYSTEMD_UNIT=kubelet.service]" }}
```

Comment:
This seems to have been executed with an error.

Reply:
Changed this and moved defaults back to values-file